bedrock/firefox/templates/firefox/whatsnew/index.id.html

```diff
@@ -25,29 +25,39 @@
 
 {% block content %}
 <main role="main" class="section">
-  <div class="content">
-    <a class="moz-logo" href="{{ url('mozorg.home') }}"><img src="{{ static('img/logos/mozilla/wordmark-dark.svg')}}" alt="Mozilla" width="101" height="34"></a>
+  <header class="main-header">
```

Comment:
Could this common header be an include that is shared between `/whatsnew` templates?

Reply:
Good suggestion on making an `@import` and an `include`. Will do.